src/app/index.ts

```diff
@@ -307,6 +314,10 @@ const oauthCallback = (req: express.Request, res: express.Response, next: expres
 };
 
 const oauthState = (req: express.Request, res: express.Response) => {
+  const isDevelopment = NODE_ENV === NodeEnvVariables.DEVELOPMENT;
```

Comment:
1. There's a different convention used to check NODE_ENV here https://github.com/onaio/express-server/blob/80fd0e62b1e2f28853784a809542e94dfa0576a4/src/app/index.ts#L134
2. The intention here is not well understood. Could you explain the problem we are looking to solve here?

Reply:
@machariamuguku this returns a 403 for this endpoint if the node_env is set to production. 

app.get('env') always defaults to `development` even when its not defined, which is a behavior I do not want. To be safer, one needs to explicitley add the node_env as `production`.

Reply:
@p-netm Why do we want to return 403 for development? Does this mean the server won't work if env is set to development? I think defaulting to development is a sane default? And setting to production when needed. That being said, if it's important then we can change both implementations?

Reply:
@p-netm we also have some instances that cause infinite refresh if NODE_ENV is set to production and this would mean they're locked out

Reply:
> @p-netm Why do we want to return 403 for development? Does this mean the server won't work if env is set to development? I think defaulting to development is a sane default? And setting to production when needed. That being said, if it's important then we can change both implementations?

we want to return 403 in production not in development. this is a security kind of thing, so the safest default should be restrictive, defaulting to `development` has the potential that when its not set in a production environment it will still expose the `oauth/state` details.

Reply:
> @p-netm we also have some instances that cause infinite refresh if NODE_ENV is set to production and this would mean they're locked out

I think that in itself is a problem that we should look at. running the code using a development build in production should not be the solution.